### ISSUE TITLE
monoallelic_Y_hemizygous 

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7497]
**Preferred term label:**
monoallelic_Y_hemizygous 

**Synonyms**
Y-linked inheritance 
HP:0001450

**Definition (free text, please give PubMed ID)**
The allelic requirement for Y-linked inheritance 

**Parent term (use hpo.jax.org/app)**
allelic requirement (requested)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as allelic requiement synonym
